infra/infra.go

```diff
@@ -42,6 +43,12 @@ func main() {
 		Short('v').
 		StringMapVar(&dr.FlagDeploymentVars)
 
+		// Storing Data on Object Store
+	Objstore := app.Command("block-sync", `Using Thanos to store the data`)
```

Comment:
There are a couple of things that need to be corrected here.

First of all, we were creating a new binary/CLI dedicated to block syncing. It would be better to crate a new tool in the tools directory and deploy it separately, only it's needed.

The other issue is the style here. `Objstore` should be `objstore`.